### PR TITLE
Use bitnami helm chart to manage etcd

### PR DESCRIPTION
**Pull request by bastjan**
## Checklist

- [x] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation.
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.

### CODE REVIEW

File: class/defaults.yml [hunk 1]
```diff
@@ -1,4 +1,43 @@
...
+      etcd:
+        source: https://charts.bitnami.com/bitnami
+        version: "9.1.0"
+
+    helm_release_name: ${_instance}
+    helm_values:
+      image: ${etcd:images:etcd}
+      volumePermissions:
+        image: ${etcd:images:os-shell}
+      auth:
```
Comment (bastjan) [inline class/defaults.yml:30]: Values are updated to remove helm's secret generation and for security by default.

File: docs/modules/ROOT/pages/references/parameters.adoc
Comment (bastjan) [inline docs/modules/ROOT/pages/references/parameters.adoc:1]: Will add some example after setting up etcd on our cluster.

File: class/etcd.yml [hunk 2]
```diff
@@ -8,4 +15,14 @@ parameters:
       - input_paths:
           - ${_base_directory}/component/main.jsonnet
         input_type: jsonnet
-        output_path: etcd/
+        output_path: etcd/${_instance}
+
+      - output_path: ${_instance}/01_etcd_helmchart
```
Comment (simu) [inline class/etcd.yml:20]: I'd prefer having a consistent output base path, either `etcd/${_instance}` or `${_instance}`

**Review by simu — APPROVED** [review]
LGTM overall, you may want to add the required secrets with sensible Vault references in the component defaults.

File: class/etcd.yml (hunk 2 above)
Comment (bastjan) [reply class/etcd.yml]: fixed 👍

**Pull request merged**